lighthouse-core/audits/byte-efficiency/legacy-javascript.js

```diff
@@ -348,18 +347,89 @@ class LegacyJavascript extends Audit {
     return urlToMatchResults;
   }
 
+  /**
+   * @param {PatternMatchResult[]} matches
+   */
+  static estimateWastedBytes(matches) {
+    const polyfillSignals = matches.filter(m => !m.name.startsWith('@')).map(m => m.name);
```

Comment:
what's the significance of `@`? mind explaining in a comment? :)

I can't really find where transformSignals are used